nestkernel/nest_names.h

```diff
@@ -407,6 +407,8 @@ extern const Name taus_eta;   //!< Specific to population point process model
 extern const Name taus_syn;   //!< Synapse time constants (array)
 extern const Name taus_rise;  //!< Synapse rise constants (array)
 extern const Name taus_decay; //!< Synapse decay constants (array)
+extern const Name E_rev;      //!< Reversal potential (array)
+extern const Name num_of_receptors; //!< number of receptor ports
```

Comment:
In other names, we have used `n_`, e.g., `n_events`.  I suggest `n_receptors` here for consistency.